docs/user/openstack/README.md

```diff
@@ -646,6 +646,49 @@ If you ran the installer with a [custom CA certificate](#self-signed-openstack-c
 oc edit configmap -n openshift-config cloud-provider-config
 ```
 
+### Enabling Octavia for Load Balancer Services
+
+As a day 2 operation, you can create load balancer service types and ingress controllers that have load balancers as backends by using octavia. However, there are a few known issues to be aware of:
+* Only TCP traffic is supported.
+* The floating IP addresses that are attached to your Octavia load balancers are not deleted when you delete your cluster.
```

Comment:
could we add a link to a BZ tracking this feature, so 1) users can follow the feature and 2) we have a BZ in our backlog and solve it one day :)

Reply:
Talked this over with Martin, I am not going to track this with a BZ because we can never backport support for this. It would be considered a feature backport. We will just document it here and downstream.